doc-content/enterprise-only/openshift/rhpam71-prod-immutable-monitor-ref.adoc

```diff
@@ -0,0 +1,333 @@
+////
+    AUTOGENERATED FILE - this file was generated via ./tools/gen_template_docs.py.
+    Changes to .adoc or HTML files may be overwritten! Please change the
+    generator or the input template (./*.in)
+////
+= rhpam71-prod-immutable-monitor
+[id='rhpam71-prod-immutable-monitor-ref']
+
+Application template for a router and monitoring console in a production environment, for Red Hat Process Automation Manager 7.1
```

Comment:
I explicitly have a request to get this to 7.1 so that's what I will do first - into master and 7.11.x and 7.4.x. Then - next week, around freeze time - I'll update with 7.2 autogenerated sections and make 7.2 PRs just to master and 7.14.x